test/argent-account-test.ts

```diff
@@ -1,59 +1,81 @@
 import { expect } from "chai";
-import { starknet } from "hardhat";
-import { StarknetContract, StarknetContractFactory, ArgentAccount } from "hardhat/types/runtime";
+import hardhat, { starknet, ArgentAccount } from "hardhat";
+import { StarknetContract, StarknetContractFactory } from "hardhat/types/runtime";
 import { TIMEOUT } from "./constants";
-import { ensureEnvVar, expectFeeEstimationStructure } from "./util";
+import { expectFeeEstimationStructure, getOZAccount, mint } from "./util";
 
 describe("Argent account", function () {
     this.timeout(TIMEOUT);
 
     let mainContractFactory: StarknetContractFactory;
     let mainContract: StarknetContract;
-    let utilContractFactory: StarknetContractFactory;
-    let utilContract: StarknetContract;
-
-    let account: ArgentAccount;
+    let argentAccountAddress: string;
+    const argentAccountPrivateKey =
+        "0x66826acbe6ab1e8612124c0cb413b17695119148aabfe010b1851a9b78ea295";
+
+    /**
+     * Returns an instance of ArgenAccount. Expected to be deployed.
+     */
+    const getArgentAccount = async () => {
```

Comment:
Seems to me that this could also be used somewhere sometime, could we also put it on `util.ts` or leave it here?

Reply:
That's how it was until this morning, then I realized that the deployment address needs to be specified through an env var, which in turn needs to be set using the set-devnet-vars.sh in the plugin CI. So I decided it's the best to keep it in the same file where the address can be obtained (without it having to be specified externally).